conf/app.routes

```diff
@@ -106,8 +106,16 @@ GET  /subscribed/amend-details/verify-email           uk.gov.hmrc.cgtpropertydis
 GET  /subscribed/amend-details/email-verified         uk.gov.hmrc.cgtpropertydisposalsfrontend.controllers.email.SubscribedChangeEmailController.emailVerified()
 
 ### Change contact name
-GET  /subscription/amend-details/enter-contact-name      uk.gov.hmrc.cgtpropertydisposalsfrontend.controllers.name.ContactNameController.enterContactName()
-POST /subscription/amend-details/enter-contact-name      uk.gov.hmrc.cgtpropertydisposalsfrontend.controllers.name.ContactNameController.enterContactNameSubmit()
+GET  /subscription/amend-details/enter-contact-name      uk.gov.hmrc.cgtpropertydisposalsfrontend.controllers.name.SubscriptionChangeContactNameController.enterContactName()
+POST /subscription/amend-details/enter-contact-name      uk.gov.hmrc.cgtpropertydisposalsfrontend.controllers.name.SubscriptionChangeContactNameController.enterContactNameSubmit()
+
+### Change subscribed contact name
+GET  /subscribed/amend-details/enter-contact-name      uk.gov.hmrc.cgtpropertydisposalsfrontend.controllers.name.SubscribedChangeContactNameController.enterContactName()
+POST /subscribed/amend-details/enter-contact-name      uk.gov.hmrc.cgtpropertydisposalsfrontend.controllers.name.SubscribedChangeContactNameController.enterContactNameSubmit()
+
+GET  /subscribed-without-id/amend-details/enter-contact-name      uk.gov.hmrc.cgtpropertydisposalsfrontend.controllers.name.SubscribedWithoutIdChangeContactNameController.enterIndividualName()
```

Comment:
is this endpoint appropriately named?

Reply:
probably not - but when we get the actual url's designed then we'll have a proper one